src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -130,3 +132,52 @@ function Push-OutputBinding {
         }
     }
 }
+
+<#
+.SYNOPSIS
+    Write the formated output of the pipeline object to the information stream before passing the object down to the pipeline.
+.DESCRIPTION
+    Write the formated output of the pipeline object to the information stream before passing the object down to the pipeline.
+.PARAMETER InputObject
+    The object from pipeline.
+.PARAMETER WriteToInformationChannel
+    Make the command write the pipeline object to the information stream.
+#>
+function Trace-PipelineObject {
+
+    [CmdletBinding(DefaultParameterSetName = "Default")]
+    param(
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [object]
+        $InputObject,
+
+        [Parameter(ParameterSetName = "WriteToInformationChannel")]
+        [switch]
+        $WriteToInformationChannel
+    )
+
+    Begin {
+        if ($PSCmdlet.ParameterSetName -eq "Default") {
+            $outStringCmd = $ExecutionContext.InvokeCommand.GetCommand("Microsoft.PowerShell.Utility\Out-String", [CommandTypes]::Cmdlet)
+            $tracePipeCmd = $ExecutionContext.InvokeCommand.GetCommand("Microsoft.Azure.Functions.PowerShellWorker\Trace-PipelineObject", [CommandTypes]::Function)
```

Comment:
Is there a reason you're doing `$ExecutionContext.InvokeCommand.GetCommand` instead of `Get-Command`?

Reply:
To avoid creating two pipelines. Method call is much faster than command execution.